src/components/Integrations/BigQuery/FreeAccount/FreeAccount.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { Fragment, useState } from 'react';
+import './styles/index.css';
+import { useIntl } from 'react-intl';
+import SafeRedirect from '../../../SafeRedirect';
+
+interface IFreeAccountProps {}
+
+const FreeAccount: React.SFC<IFreeAccountProps> = (props) => {
+  const intl = useIntl();
+  const _ = (id: string, values: any) => intl.formatMessage({ id: id }, values);
+
+  const [redirect, setRedirect] = useState(false);
+
+  const tittle = (
+    <div>
+      <span className="data-studio-logo"></span>
+      <h1>{_('big_query.free_title', '')}</h1>
+    </div>
+  );
+
+  const list = (
+    <div>
+      <ul className="dp-list-detail">
+        <li>
+          <span className="check-icon"></span>
+          <span className="list-item-span">{_('big_query.free_ul_item_one', '')}</span>
+        </li>
+        <li className="list-item">
+          <span className="check-icon"></span>
+          <span className="list-item-span">{_('big_query.free_ul_item_two', '')}</span>
+        </li>
+      </ul>
+    </div>
+  );
+
+  const btn = (
+    <button
+      type="button"
+      className="dp-button button-medium primary-green"
+      onClick={() => setRedirect(true)}
+    >
+      {_('big_query.free_btn_redirect', '')}
+    </button>
+  );
+
+  if (redirect) {
+    return <SafeRedirect to="/ControlPanel/AccountPreferences/UpgradeAccount" />;
+  }
+
+  return (
+    <Fragment>
```

Comment:
We usually wrap the code between <></> when we have more than one element inside. In this case I think that it's not necessary because you only have one section.